src/components/HeaderForm.js

```diff
@@ -948,25 +968,45 @@ class HeaderForm extends Component {
                   region={this.state.region}
                 />
               )}
+              
               {customFilesFlag && (
-                <div style={{ display: "flex" }}>
-                  {DataPositionFormRowComponent}
+                <div className="d-flex justify-content-between align-items-start">
+                  <div>
+                    {DataPositionFormRowComponent}
+                  </div>
+                  <div className="d-flex justify-content-end align-items-start flex-shrink-0">
+                  {!readsExist(this.state.tracks) && (
+                    <>
+                      <Button
+                        onClick={this.openPopup}
+                        outline
+                        active={this.state.simplify || this.state.removeSequences}
+                      >
+                      <FontAwesomeIcon icon={faGear} /> Simplify
+                      </Button>
+                      <PopupDialog open={this.state.popupOpen} close={!this.openPopup} width="400px">
```

Comment:
I think `openPopup()` maybe needs to be renamed to `togglePopup()`? And you would just pass it here? It looks like you are trying to negate the function and I don't think this will actually work if you run it and try and open the dialog (so maybe that warrants a test?).

```suggestion
                      <PopupDialog open={this.state.popupOpen} close={this.togglePopup} width="400px">
```